ui/appui-react/src/appui-react/layout/widget/widget-content-layout/header/HeaderFirstRow.scss

```diff
@@ -0,0 +1,27 @@
+.nz-widget-widgetContentLayout-header-headerFirstRow {
+  gap: var(--iui-size-s);
+  justify-content: space-between;
+  display: grid;
+  grid-template-columns: auto 1fr;
+  width: 100%;
+
+  &.nz-singleColumn {
+    grid-template-columns: 1fr;
+  }
+
+  .nz-leftContent {
+    display: flex;
+    gap: var(--iui-size-s);
+    align-items: center;
+    text-wrap-mode: nowrap;
+    overflow: auto;
+  }
+}
+
+@container header-layout (width < 200px) {
```

Comment:
Is this expected to be used by consumers? If not, maybe we can remove the name? If not, maybe we should prefix it with `nz` to avoid ambiguity?